sdk/provisioning/Azure.Provisioning.AppService/tests/BasicLiveAppServiceTests.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.AppService.Tests;
+
+public class BasicLiveAppServiceTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true */)
+{
+    [Test]
+    [Description("https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.web/function-app-create-dynamic/main.bicep")]
+    public async Task CreateBasicFunctionApp()
```

Comment:
lets move the LIveOnly attribute to the tests so its easier to see from the code what is live vs running in ci